examples/README.md

```diff
@@ -17,6 +18,9 @@ Example: [Using Incremental Learning Job in Helmet Detection Scenario](./increme
 ### Federated Learning
 Example: [Using Federated Learning Job in Surface Defect Detection Scenario](./federated_learning/surface_defect_detection/README.md)
 
+### Lifelong Learning
+Example: [Using Lifelong Learning Job in Thermal Comfort Prediction Scenario](./lifelong_learning/atcii/README.md)

```

Comment:
![image](https://user-images.githubusercontent.com/1785240/129684072-bdccf72e-092a-4211-86b7-ad74deffe16d.png)

Some of the hyperlinks in this document are missing.

Reply:
check & fixed